File: iteration3/src/SelectionProblem.py
class SelectionProblem:
    def __init__(self, id, not_registered_course, description):
        self._not_registered_course = not_registered_course
        self._id = id
        self._description = description

    def get_id(self):
        return self._id

    def get_not_registered_course(self):
        return self._not_registered_course

    def get_description(self):
        return self._description

File: iteration2/src/CSE3063F22P1_GRP9/SelectionProblem.java
package CSE3063F22P1_GRP9;

public class SelectionProblem {
	private int Id;
	private Course notRegisteredCourse;
	private String description;
	
	public SelectionProblem(Course notRegisteredCourse){
		this.notRegisteredCourse = notRegisteredCourse;
	}
	
	public SelectionProblem(int Id,Course notRegisteredCourse,String description){
		this.Id = Id;
		this.notRegisteredCourse = notRegisteredCourse;
		this.description = description;
	}
	
	public int getId() {
		return Id;
	}

	public Course getNotRegisteredCourse() {
		return notRegisteredCourse;
	}

	public String getDescription() {
		return description;
	}
	
}